card/lib/card/format/error.rb

```diff
@@ -2,35 +2,34 @@ class Card
   class Format
     module Error
       def rescue_view e, view
-        raise e if Rails.env =~ /^cucumber|test$/
-        if focal?
-          focal_error e, view
-        else
-          # TODO: consider rendering dynamic error view here.
-          nested_error e, view
-        end
-      end
-
-      def debug_error e
-        raise e if Card[:debugger]&.content == "on"
```

Comment:
I missed that you removed this but I don't want to miss this feature. It was the shining light that guided me through the hell of nasty bugs. Now I'm lost in the darkness of ugly formatted logger messages.